typescript/infra/config/environments/mainnet3/warp/configGetters/utils.ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+
+import { ChainMap, ChainName, MovableTokenConfig } from '@hyperlane-xyz/sdk';
+import { arrayToObject, objMap } from '@hyperlane-xyz/utils';
+
+import { getRegistry } from '../../../../registry.js';
+import { WarpRouteIds } from '../warpIds.js';
+
+const REBALANCER = '0xa3948a15e1d0778a7d53268b651B2411AF198FE3';
+
+type RebalancingConfig = Required<
+  Pick<MovableTokenConfig, 'allowedRebalancingBridges' | 'allowedRebalancers'>
+>;
+
+export function getRebalancingBridgesConfigFor(
```

Comment:
we should name this appropriately to indicate this only works for USDC
is your intention to add more bridges here later? eg everclear?